frontend/src/static/js/v_summary.js

```diff
@@ -137,7 +137,7 @@ window.vSummary = {
       this.getFiltered();
     },
     filterBreakdown() {
-      this.getFiltered();
+      this.setSummaryHash();
```

Comment:
Hmm, is it necessary to have a watcher? How about just make it activate on click or something?

https://github.com/reposense/RepoSense/blob/bf2fa03a34da18fc89112efc1c2ef544b716c699/frontend/src/index.pug#L133-L134

Reply:
It is not so necessary I believe.